components/brave_shields/common/features.cc

```diff
@@ -10,6 +10,8 @@
 namespace brave_shields {
 namespace features {
 
+const base::Feature kBraveAdblockCnameUncloaking{
```

Comment:
good style is to write short comments for all features

Reply:
added